### PR TITLE
Deploy knative monitoring on postsubmit

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

**Comment by chaodaiG** [comment]
/hold
Let's not merge it on Friday afternoon

**Review by michelle192837 — APPROVED** [review]

File: prow/knative/Makefile [hunk 1]
```diff
@@ -29,3 +29,7 @@ deploy: get-cluster-credentials
 .PHONY: deploy-build
 deploy-build: get-build-cluster-credentials
 	kubectl apply -f ./cluster/build/
+
+.PHONY: deploy-monitoring
```
Comment (e-blackwelder) [inline prow/knative/Makefile:33]: I didn't know you could add `.PHONY` multiple times. Neat!

**Review by e-blackwelder — APPROVED** [review]

**Comment by fejta** [comment]
Is this ready to merge?

**Pull request merged**